EIPS/eip-1.md

```diff
@@ -168,37 +248,45 @@ if neither the email address nor the GitHub username are given.
 
 At least one author must use a GitHub username, in order to get notified on change requests and have the capability to approve or reject them.
 
-### `discussions-to` header
+#### `discussions-to` header
```

Comment:
I've seen a few RFCs that have been moved through the process utilizing alternative channels of communication such as Telegram and Discord essentially subverting the intent of this header. Would it be useful to allow this to link to where the group is actually communicating to be more conducive for open communication? For example, one I'd really like to see done is to be able to link to an alternative GH repository where a spec is actually iterated on. The value of this is that it allows for separate issue tracking to make sure changes are getting implemented in the spec and Async debate to occur. Furthermore, it can link to authoritative places of discussion such as a Discord server or a Zoom link where synchronous discussions regularly occur via the Readme so that the discussions can be more transparent.

If we move in this direction we can also iterate on this process further so that the concept of Rings can be more formalized (e.g. form a ETH-Core-Devs org to iterate on Core specs) such that it makes it more known where the peer review and the experts are actually communicating.

Reply:
I've also seen cases where the Magician's link points off to other places where discussion is happening.  Does that suffice?

Reply:
Still, I agree that I'd rather see the bulk of the discussion happen here or on the Magicians where the history can be maintained.